src/providers/object-provider.js

```diff
@@ -208,7 +208,7 @@ export default class YamcsObjectProvider {
         }
 
         if (isAggregate) {
-            obj.type = 'folder';
+            obj.type = ;
```

Comment:
Something doesn't seem right here...

Reply:
Agreed! That's an odd one. Not sure how I would have missed that.